public/sass/_angular.scss

```diff
@@ -1981,3 +1981,51 @@ $easing: cubic-bezier(0, 0, 0.265, 1);
     color: $text-muted;
   }
 }
+
+.grafana-info-box {
```

Comment:
@ashharrison90 can't we move them to Global emotion instead so we can get rid of the scss (eventually)

Reply:
@torkelo yeah i've done that in some cases. but i'm also trying to just straight remove styles that i don't think provide any value anymore. in this case, we have a drop-in replacement with the `<Alert severity="info" />` component, so i've adjusted the existing uses and moved these styles to the `angular` SCSS file.

the intention here is to collect the styles that are now only used by angular code, so that when we remove angular in grafana 12 we can just remove this SCSS file as well and be done with it. it also gives a good timeline for external plugins to know when they have to move off using them. i guess we could put these in global emotion, but we've seen some performance problems when adding more and more global styles (see https://github.com/grafana/grafana/pull/89687), so i'd rather only do it if they're going to stay around. wdyt? 🤔

Reply:
> there seems to also be some sort of style leak here as dragging the news drawer back and forth seems to continuously add styles to the

interesting :)

Reply:
btw we've fixed that by moving the dynamic styles into a style tag: https://github.com/grafana/grafana/pull/89691

which also matches the [emotion best practices doc](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles)

normally not an issue, but interesting to keep in mind for those things that are _really_ dynamic 😄